tests/unit/utils/test_gitfs.py

```diff
@@ -5,15 +5,25 @@
 '''
 
 # Import python libs
-from __future__ import absolute_import, unicode_literals, print_function
+from __future__ import absolute_import, print_function, unicode_literals
 
-# Import Salt Testing libs
-from tests.support.unit import TestCase
-from tests.support.mock import MagicMock, patch
+import os
+import shutil
+from time import time
+
+import pygit2
```

Comment:
This import have to be surrounded with try-except too to protect test run on systems lacking pygit2.

Reply:
There are test systems without it? It would be really great to have a quicker way to find out :) I'll fix that as well.

Reply:
I used the HAS_PYGIT2 constant now to decide on the import.